scripts/get-ui-templates.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * All the UI templates used by 3rd-party CLIs.
+ *
+ */
+const getUiTemplates = () => [
+  '@coveo/cra-template',
+  '@coveo/vue-cli-plugin-typescript',
+  '@coveo/angular',
+];
+
```

Comment:
It's the only place where we have to hardcode the ui templates that we need to publish for  E2E tests.